config/clients/dotnet/template/README_calling_api.mustache

```diff
@@ -208,16 +211,9 @@ var response = await {{appCamelCaseName}}Api.ReadChanges(type, pageSize, continu
 
 ```csharp
 var body = new ListObjectsRequest{
-    AuthorizationModelId = "01GAHCE4YVKPQEKZQHT2R89MQV",
-    User = "anne",
-    Relation = "can_read",
-    Type = "document",
```

Comment:
Can we keep the contextual tuples here?

Just so people know they exist and work in case they missed that in the docs?

Reply:
I will add context tuple in, but will use a known relationship/type so that people can just paste it in and it still works.